### ISSUE TITLE
Material-UI datePicker

### ISSUE THREAD

**Issue by Timonzimm** [kadirahq/meteor-react-layout#23]
How would one implement a solution for the Material UI DatePicker with this package? When using the simply ReactLayout.renader it seems to make it work in a weird manner, probably due to the fact that the datePicker instance creates a new body tag.

``` javascript
let {DatePicker} = mui;

MainLayout = React.createClass({
    // Required by Material UI
    childContextTypes: {
        muiTheme: React.PropTypes.object
    },
    getChildContext() {
        return {
            muiTheme: mui.Styles
                .ThemeManager
                .getMuiTheme(mui.Styles.LightRawTheme)
        };
    },
    render: function() {
        return (
            <DatePicker hintText="Portrait Dialog" />
        );
    }

});
```

![2015-10-16 09_23_09-localhost_3000](https://cloud.githubusercontent.com/assets/5808319/10535951/8b26fc50-73e7-11e5-84c2-6c26b149c701.png)
![DatePicker issue](https://cloud.githubusercontent.com/assets/5808319/10535908/1a07245a-73e7-11e5-9ab3-c65381a8cd26.png)

**Comment by arunoda** [comment]
Could you update react-layout to the latest version. We just fixed a bug.
See: #22

**Comment by Timonzimm** [comment]
It  seems that the issue creating multiple body tag is indeed solved. But the issue with ReactLayout and datePicker is still there. I still have my last screenshot as result.

**Comment by arunoda** [comment]
Could you send me a repo with the issue and the desired result?

**Comment by Timonzimm** [comment]
A full rebuild of the project after you fixed issue #22 did the job, simple update didn't make it work properly though, sorry for the inconvenience.

**Comment by arunoda** [comment]
Great.
